Makefile

```diff
@@ -8,3 +9,15 @@ tools:
 generate:
 	docker build ./scripts/generate -t ghcr.io/liatrio/rode-api-generate:latest
 	docker run -it --rm -v $$(pwd):/rode-api ghcr.io/liatrio/rode-api-generate:latest
+
+fmtcheck:
+	lineCount=$(shell gofmt -l -s $(GOFMT_FILES) | wc -l | tr -d ' ') && exit $$lineCount
```

Comment:
exiting with the number of formatting errors. That's a good one

Reply:
stole it from https://github.com/mrparkers/terraform-provider-keycloak/blob/master/makefile :smile: